examples/HorizontalReflector1D.py

```diff
@@ -102,9 +102,9 @@
 
     result = invalg(shots, initial_value, nsteps, verbose=True, status_configuration=status_configuration)
 
-    print '...run time:  {0}s'.format(time.time()-tt)
+    print('...run time:  {0}s'.format(time.time()-tt))
 
-    obj_vals = np.array([v for k,v in invalg.objective_history.items()])
+    obj_vals = np.array([v for k,v in list(invalg.objective_history.items())])
```

Comment:
This should still work, correct?  in Python 3, `dict.items()` should be an iterator, which should be valid in a list comprehension.